pkg/testcommon/testcommon.go

```diff
@@ -408,6 +410,12 @@ func GetCachedArchive(_, _, internalExtractionPath, sourceURL string) (string, s
 		err := util.DownloadFile(archiveFullPath, sourceURL, false, "")
 		if err != nil {
 			_ = os.RemoveAll(archiveFullPath)
+			if errors.Is(err, context.DeadlineExceeded) {
```

Comment:
We'd think this wouldn't be necessary, but maybe it will prevent a test failure or two.